docs/docs/platform/activity-feed.md

```diff
@@ -7,7 +7,8 @@ sidebar_position: 1
 Novu provides a notification activity feed that monitors every outgoing message associated with its relevant metadata.
 This can be used to monitor activity and discover potential issues with a specific provider or a channel type.
 
-[Activity Screen](/img/activity.png)
+![Activity Screen](https://user-images.githubusercontent.com/58129377/194726971-52f98e3b-acdd-411c-9d5d-ca28960b8288.png)
```

Comment:
```suggestion
![Activity Screen](/img/activity.png)
```

this should actually work, and is the preferred way to do. Since the images are stored in the docosaurus assets folder. They won't render from GitHub MD viewer, and this is the expected behaviour. You can see if it worked by running the docs locally on your machine.

Wdyt?

Reply:
Yes, that is preferable. Viewing in GitHub MD Viewer was my second priority. If it's fine then, we can proceed with it.